test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -1781,6 +1781,19 @@ BOOST_AUTO_TEST_CASE(transfer_ether)
 	ABI_CHECK(callContractFunction("b(address,uint256)", oogRecipient, 10), encodeArgs());
 }
 
+BOOST_AUTO_TEST_CASE(blockhash_global_level)
+{
+	char const* sourceCode = R"(
+		contract Test {
+			function a() public returns (bytes32) {
+				return blockhash(0);
+			}
+		}
+	)";
+ 	compileAndRun(sourceCode);
+	BOOST_CHECK_EQUAL(callContractFunction("a()").empty(), false);
```

Comment:
Please use `ABI_CHECK(callContractFunction(...), encodeArgs(0))`

Reply:
Ah sorry, I misunderstood this. I think it is easier to read if you just use
`BOOST_CHECK(!callContractFunction("a()").empty())`, or perhaps even `BOOST_CHECK_EQUAL(callContractFunction("a()").length(), 32);`